src/openforms/registrations/contrib/objects_api/submission_registration.py

```diff
@@ -273,9 +274,11 @@ class ObjectsAPIV1Handler(ObjectsAPIRegistrationHandler[RegistrationOptionsV1]):
 
     @staticmethod
     def get_payment_context_data(submission: Submission) -> dict[str, Any]:
+        price = submission.price
```

Comment:
Is there a possibility for the submission to be `None` here?

Reply:
no :) otherwise the type hint should have been `Submission | None`, but the previous code already used `submission.payments` so that would've been broken in the first place